src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/process-form.service.ts

```diff
@@ -37,6 +39,27 @@ export class ProcessFormService {
         }
     }
 
+    private processPipeCommandControl(formGroup: FormGroup): void {
+        const form = formGroup.value as EditFormType;
+        if (form.prepare && form.prepare.length > 0) {
+            form.prepare.map((item) => {
+                if (item.kind === "pipe" && typeof item.command === "string") {
+                    item.command = item.command.trim().split(" ");
```

Comment:
This is clearly not enough because of quoting.
Try to look for a ready library that would parse the command line correctly, taking quoting rules into account.